player_comparator.py

```diff
@@ -72,11 +54,11 @@ def _build_players_list_from_unique_ids(self, play_by_play_dataframe, unique_ids
 
         for player_id in unique_ids:
 
-            if len(pbp[pbp['PLAYER1_ID'] == player_id]['PLAYER1_NAME'].unique()) == 1:
+            if len(pbp[pbp['player1_id'] == player_id]['player1_name'].unique()) == 1:
                 full_name, first_name, last_name = self._extract_player_name(player_id, 1, pbp)
-            elif len(pbp[pbp['PLAYER2_ID'] == player_id]['PLAYER2_NAME'].unique()) == 1:
+            elif len(pbp[pbp['player2_id'] == player_id]['player2_name'].unique()) == 1:
                 full_name, first_name, last_name = self._extract_player_name(player_id, 2, pbp)
-            elif len(pbp[pbp['PLAYER3_ID'] == player_id]['PLAYER3_NAME'].unique()) == 1:
+            elif len(pbp[pbp['player3_id'] == player_id]['player3_name'].unique()) == 1:
```

Comment:
Create a constants.py file that contains all the magic strings as variables.

E.g. 

```
class PlayerKeys:
     player3_id = 'player3_id'
```

Reply:
This will reduce the chance of typos